exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -244,7 +244,13 @@ def _translate_to_prometheus(
             for pre_metric_family_id, label_values, value in zip(
                 pre_metric_family_ids, label_valuess, values
             ):
-                if isinstance(metric.data, Sum):
+                is_non_monotonic_sum = isinstance(metric.data, Sum) and metric.data.is_monotonic == False
+                is_cumulative = isinstance(metric.data, Sum) and metric.data.aggregation_temporality == AggregationTemporality.CUMULATIVE
```

Comment:
I don't think you need this cumulative check. The preferred temporality for this exporter is explicitly typed with CUMULATIVE. You can assume points are always cumulative.

Reply:
noted — my intent was to reflect the spec as closely as possible in code. if including this line wouldn't cause problems, i'd advocate to keep it, just for the sake of being explicit.

Reply:
that said, i could also add a comment explaining that the code assumes the temporality is explicit

Reply:
I don't seen an issue with keeping it explicit